composer.json

```diff
@@ -31,7 +31,7 @@
     "require-dev": {
         "symfony/yaml": "~2.8 || ~3.4 || ~4.2 || ~5.0",
         "phpunit/php-code-coverage": "^2.2|^3.0",
```

Comment:
Any particular reason one is | and the other is ||?

Reply:
The composer normalizer changes it to `||`.